marlowe-dashboard-client/src/Marlowe/Client.purs

```diff
@@ -20,6 +20,13 @@ import Marlowe.Semantics (Input, MarloweData, MarloweParams, Slot, TransactionEr
 import Plutus.Contract.StateMachine (InvalidTransition, SMContractError)
 import Wallet.Types (ContractError)
 
+-- These are the endpoints of the main marlowe (control) contract.
+-- TODO: provide encoding for these and use that with the `invokeEndpoint` call in Capability.Marlowe
```

Comment:
Could you explain what you mean by this TODO a little more? (And then update the TODO 😃)

Reply:
Will do...